packages/core/src/integrations/third-party-errors-filter.ts

```diff
@@ -32,6 +33,13 @@ interface Options {
     | 'drop-error-if-exclusively-contains-third-party-frames'
     | 'apply-tag-if-contains-third-party-frames'
     | 'apply-tag-if-exclusively-contains-third-party-frames';
+
+  /**
+   * @experimental
+   * If set to true, the integration will exclude frames that are internal to the Sentry SDK from the third-party frame detection.
+   * Note that enabling this option might lead to errors being misclassified as third-party errors.
+   */
+  experimentalExcludeSentryInternalFrames?: boolean;
```

Comment:
l/nit (feel free to disgredard): Should we rather call this 

```suggestion
  experimentalIgnoreSentryInternalFrames?: boolean;
```

? IIUC we want to avoid making decisions based on these frames so we ignore them. But really up to you, I'm fine with either.